# Statistics Case Study II Solutions

## Problem Statement 1:

In each of the following situations, state whether it is a correctly stated hypothesis testing problem and why?

#### 1. 𝐻0: 𝜇 = 25, 𝐻1: 𝜇 ≠ 25

Solution:
The stated hypothesis testing problem is CORRECT.

H0 is called as null hypothesis. The null hypothesis claims that the initial statement is assumed to be true.

H1 is called as alternate hypothesis. The alternate hypothesis always claims against the null hypothesis. 

#### 2. 𝐻0: 𝜎 > 10, 𝐻1: 𝜎 = 10

Solution:
The stated hypothesis testing problem is INCORRECT.

H0, the null hypothesis will claims to be equality. But in some cases like, when alternate hypothesis is stated with < sign, the null hypothesis will be considered as >= sign and when alternate hypothesis is stated with > sign, the null hypothesis will be considered as <= sign. So above statement is not correct.

#### 3. 𝐻0: 𝑥̅ = 50, 𝐻1: 𝑥̅ ≠ 50

Solution:
The stated hypothesis testing problem is INCORRECT.

Hypothesis statements are for entire population or distribution

#### 4. 𝐻0: 𝑝 = 0.1, 𝐻1: 𝑝 = 0.5

Solution: The stated hypothesis testing problem is INCORRECT.

H0, the null hypothesis will claims to be equality. But in some cases like, when alternate hypothesis is stated with < sign, the null hypothesis will be considered as >= sign and when alternate hypothesis is stated with > sign, the null hypothesis will be considered as <= sign. 

Also Values on both the hypothesis are different and hypothesis testing doesn't work on probability.

#### 5. 𝐻0: 𝑠 = 30, 𝐻1: 𝑠 > 30

Solution:
The stated hypothesis testing problem is INCORRECT.

Hypothesis statements are for entire population or distribution. Samples are not considered under hypothesis testing.

## Problem Statement 2:

The college bookstore tells prospective students that the average cost of its textbooks is Rs. 52 with a standard deviation of Rs. 4.50. A group of smart statistics students thinks that the average cost is higher. To test the bookstore’s claim against their alternative, the students will select a random sample of size 100. Assume that the mean from their random sample is Rs. 52.80. Perform a hypothesis test at the 5% level of significance and state your decision.

#### Solution:

Please refer files\PS_2_Solution_image.png

![PS_3_Solution_image.png](files\PS_2_Solution_image.png)

## Problem Statement 3:

A certain chemical pollutant in the Genesee River has been constant for several years with mean μ = 34 ppm (parts per million) and standard deviation σ = 8 ppm. A group of factory representatives whose companies discharge liquids into the river is now claiming that they have lowered the average with improved filtration devices. A group of environmentalists will test to see if this is true at the 1% level of significance. Assume \ that their sample of size 50 gives a mean of 32.5 ppm. Perform a hypothesis test at the 1% level of significance and state your decision.

#### Solution:

please refer files\PS_3_Solution_image.png

![PS_3_Solution_image.png](files\PS_3_Solution_image.png)

## Problem Statement 4:

Based on population figures and other general information on the U.S. population, suppose it has been estimated that, on average, a family of four in the U.S. spends about
$1135 annually on dental expenditures. Suppose further that a regional dental association wants to test to determine if this figure is accurate for their area of country. To test this,
22 families of 4 are randomly selected from the population in that area of the country and a log is kept of the family’s dental expenditure for one year. The resulting data are given below. Assuming, that dental expenditure is normally distributed in the population, use the data and an alpha of 0.5 to test the dental association’s hypothesis.
1008, 812, 1117, 1323, 1308, 1415, 831, 1021, 1287, 851, 930, 730, 699,
872, 913, 944, 954, 987, 1695, 995, 1003, 994

#### Solution:

Given:

Mean = 1135

Sample size(n) = 22

Alpha = 0.05

please refer files\PS_4_Solution_image_1.png

![PS_4_Solution_image_1.png](files\PS_4_Solution_image_1.png)

In [1]:
import statistics as st
import math

sample = [1008, 812, 1117, 1323, 1308, 1415, 831, 1021, 1287, 851, 930, 730, 699, 872, 913, 944, 954, 987, 1695, 995, 1003, 994]
sample_mean = st.mean(sample)
sample_mean

1031.3181818181818

please refer files/PS_4_Solution_image_2.png

![PS_4_Solution_image_2.png](files/PS_4_Solution_image_2.png)

In [2]:
val = []
[val.append((i - sample_mean) ** 2) for i in sample]
sample_std_dev = math.sqrt(sum(val)/21)

please refer files/PS_4_Solution_image_3.png

![PS_4_Solution_image_3.png](files/PS_4_Solution_image_3.png)

In [3]:
t = (sample_mean - 1135) / (sample_std_dev / math.sqrt(22))
t

-2.023137479931484

please refer files/PS_4_Solution_image_4.png

![PS_4_Solution_image_4.png](files/PS_4_Solution_image_4.png)

## Problem Statement 5:

In a report prepared by the Economic Research Department of a major bank the
Department manager maintains that the average annual family income on Metropolis is
$48,432. 

What do you conclude about the validity of the report if a random sample of 400
families shows and average income of $48,574 with a standard deviation of 2000?

#### Solution:

##### Given:

Population Mean = 48432

Sample Size = 400

Sample Mean = 48574

Sample Standard Deviation = 2000

Please refer files/PS_4_Solution_image_3.png

![PS_4_Solution_image_3.png](files/PS_4_Solution_image_3.png)

In [4]:
sample_mean = 48574
population_mean = 48432
sample_std_dev = 2000
n = 400

t = (sample_mean - population_mean) / (sample_std_dev / math.sqrt(400))
t

1.42

please refer files/PS_5_Solution_image.png

![PS_5_Solution_image.png](files/PS_5_Solution_image.png)

## Problem Statement 6:

Suppose that in past years the average price per square foot for warehouses in the United
States has been $32.28. 

A national real estate investor wants to determine whether that
figure has changed now. The investor hires a researcher who randomly samples 19
warehouses that are for sale across the United States and finds that the mean price per
square foot is $31.67, with a standard deviation of $1.29. assume that the prices of
warehouse footage are normally distributed in population. If the researcher uses a 5%
level of significance, what statistical conclusion can be reached? What are the
hypotheses?

#### Solution:

##### Given:

population mean = 32.28

n = 19

sample mean = 31.67

sample std dev = 1.29

alpha = 0.05

In [5]:
pleas(31.67-32.28)/(1.29/math.sqrt(19))

-2.06118477175179

please refer files/PS_6_Solution_image.jpg

![PS_6_Solution_image.jpg](files/PS_6_Solution_image.jpg)

## Problem Statement 7:

Please refer files/PS_7.png

![PS_7.png](files/PS_7.png)

#### Solution:

Please refer files/PS_7_Solution_image_1.png

![PS_7_Solution_image_1.png](files/PS_7_Solution_image_1.png)

please refer files/PS_7_Solution_image_2.png

![PS_7_Solution_image_2.png](files/PS_7_Solution_image_2.png)

please share files/PS_7_Solution_image_3.png

![PS_7_Solution_image_3.png](files/PS_7_Solution_image_3.png)

please refer files/PS_7_Solution_image_4.png

![PS_7_Solution_image_4.png](files/PS_7_Solution_image_4.png)

please refer files/PS_7_Solution_image_5.jpg
![PS_7_Solution_image_5.jpg](files/PS_7_Solution_image_5.jpg)

please refer files/PS_7_Solution_image_6.jpg

![PS_7_Solution_image_6.jpg](files/PS_7_Solution_image_6.jpg)

Please refer files/PS_7_Solution_image_7.jpg)


![PS_7_Solution_image_7.jpg](files/PS_7_Solution_image_7.jpg)

please refer files/PS_7_Solution_image_8.jpg

![PS_7_Solution_image_8.jpg](files/PS_7_Solution_image_8.jpg)

#### Conclusions
1. The size of the critical region, and consequently the probability of a type I error ,
can always be reduced by appropriate selection of the critical values.
2. Type I and type II errors are related. A decrease in the probability of one type of error
always results in an increase in the probability of the other, provided that the sample
size n does not change.
3. An increase in sample size will generally reduce both alpha and beta, provided that the
critical values are held constant.
4. When the null hypothesis is false, beta increases as the true value of the parameter
approaches the value hypothesized in the null hypothesis. The value of $$\beta decreases
as the difference between the true mean and the hypothesized value increases.

## Problem Statement 8:

Find the t-score for a sample size of 16 taken from a population with mean 10 when the
sample mean is 12 and the sample standard deviation is 1.5

#### Solution:

please refer files/PS_8_Solution_image.png

![PS_8_Solution_image.png](files/PS_8_Solution_image.png)

## Problem Statement 9:

Find the t-score below which we can expect 99% of sample means will fall if samples of
size 16 are taken from a normally distributed population.

PLease refer files/PS_9_Solution_image.jpg

![PS_9_Solution_image.jpg](files/PS_9_Solution_image.jpg)

## Problem Statement 10:

If a random sample of size 25 drawn from a normal population gives a mean of 60 and a
standard deviation of 4, find the range of t-scores where we can expect to find the
middle 95% of all sample means. Compute the probability that 𝑃(−𝑡0.05 < 𝑡 < 𝑡0.10).

Please refer files/PS_10_Solution_image.jpg

![PS_10_Solution_image.jpg](files/PS_10_Solution_image.jpg)

### Problem Statement 11:

Please refer files/PS_11.png

![PS_11.png](files/PS_11.png)

### Solution:

H0 : μ1 – μ2 = 0

H1 : μ1 – μ2 ≠ 0

please refer files/Z_Formula.png

![Z_Formula.png](files/Z_Formula.png)

In [1]:
import math

In [2]:
Z = ((452 - 523) - 0)/math.sqrt((math.pow(212,2)/1200) + (math.pow(185,2)/800))
Z

-7.926428526759299

p-value: p(z < -7.926) = 0

So this concludes that the null hypothesis (H0) can be rejected. The number of people travelling from Bangalore to Chennai is not same as the people travelling from Bangalore to Hosur.

### Problem Statement 12:

Is there evidence to conclude that the number of people preferring Duracell battery is
different from the number of people preferring Energizer battery, given the following:

Population 1: Duracell

n1 = 100

x1 = 308

s1 = 84

Population 2: Energizer

n2 = 100

x2 = 254

s2 = 67

### Solution:

H0 : μ1 – μ2 <= 45

H1 : μ1 – μ2 > 45

Please refer files/Z_Formula.png

![Z_Formula.png](files/Z_Formula.png)

In [3]:
Z = ((308 - 254) - 45)/math.sqrt((math.pow(84,2)/100) + (math.pow(67,2)/100))
Z

0.837617111389407

p-value: p(z > 0.8376) = 1 - p(z < 0.84) = 1 - 0.7995 = 0.2005

p-value is less than z value. So we null hypothesis (H0) will be rejected at any level of significance.

### Problem Statement 13:

Please refer files/PS_13.png

![PS_13.png](files/PS_13.png)

### Solution:

H0 : μ1 – μ2 = 0

H1 : μ1 – μ2 ≠ 0

please refer files/t_Formula.png


![t_Formula.png](files/t_Formula.png)

In [4]:
t = ((0.317 - 0.21) - 0)/math.sqrt(((((14-1)*math.pow(0.12,2))+((9-1)*math.pow(0.11,2)))/(14+9-2))*((1/14)+(1/9)))
t

2.15355322387416

Critical points of t for 2 tail tests.

t(0.05) = 2.08, t(0.10) = 1.721

So we can reject null hypothesis (H0) at the 10% level of significance.

## Problem Statement 14:

The manufacturers of compact disk players want to test whether a small price reduction
is enough to increase sales of their product. Is there evidence that the small price
reduction is enough to increase sales of compact disk players?

Population 1: Before reduction

n1 = 15

x1 = Rs. 6598

s1 = Rs. 844

Population 2: After reduction

n2 = 12

x2 = RS. 6870

s2 = Rs. 669

### Solution:

H0 : μ1 – μ2 <= 0

H1 : μ1 – μ2 > 0

Please refer files/t_Formula.png

![t_Formula.png](files/t_Formula.png)

In [5]:
t = ((6598 - 6870) - 0)/math.sqrt(((((15-1)*math.pow(844,2))+((12-1)*math.pow(669,2)))/(15+12-2))*((1/15)+(1/12)))
t

-0.9098300343990461

alpha = 10% = 0.10

t critical value at 0.10 is ±1.316

So null hypothesis (H0) will not be rejected at the 10% level of significance.

## Problem Statement 15:

Comparisons of two population proportions when the hypothesized difference is zero.

Carry out a two-tailed test of the equality of banks’ share of the car loan market in 1980
and 1995.

Population 1: 1980

n1 = 1000

x1 = 53

𝑝̂ 1 = 0.53

Population 2: 1985

n2 = 100

x2 = 43

𝑝̂ 2= 0.53

### Solution:

H0 : p1 – p2 = 0

H1 : p1 – p2 ≠ 0

please refer files/Z_Formula_Probability.png

![Z_Formula_Probability.png](files/Z_Formula_Probability.png)

In [6]:
P = (53 + 43)/(1000 + 100)
P

0.08727272727272728

In [7]:
Z = ((0.53 - 0.43) - 0)/math.sqrt(P * (1-P) * ((1/1000) + (1/100)))
Z

3.378266212559581

Z critical value at 95% level of significance = 1.645

Z critical value at 99% level of significance = 2.33

So this concludes, the null hypothesis (H0) can be rejected even at 99% level of significance.

## Problem Statement 16:

Carry out a one-tailed test to determine whether the population proportion of traveler’s
check buyers who buy at least $2500 in checks when sweepstakes prizes are offered as
at least 10% higher than the proportion of such buyers when no sweepstakes are on.

Population 1: With sweepstakes

n1 = 300

x1 = 120

𝑝̂ 1 = 0.40

Population 2: No sweepstakes

n2 = 700

x2 = 140

𝑝̂ 2= 0.20

### Solution:

H0 : p1 – p2 < 0.10

H1 : p1 – p2 ≥ 0.10

##### files/Z_Formula_Probability_1.png

![Z_Formula_Probability_1.png](files/Z_Formula_Probability_1.png)

In [8]:
Z = ((0.40 - 0.20) - 0.10)/math.sqrt(((0.4 * (1-0.4))/300) + ((0.2 * (1-0.2))/700))
Z

3.1180478223116177

alpha = 0.001

Z critical value at 0.001 level of significance is 3.09. So null hypothesis (H0) can be rejected at any level of significance.

## Problem Statement 17:

A die is thrown 132 times with the following results:

Number turned up: 1, 2, 3, 4, 5, 6

Frequency: 16, 20, 25, 14, 29, 28

Is the die unbiased? Consider the degrees of freedom as n − 1.

### Solution:

H0: Die is unbaised.

H1: Die is baised.

##### files/PS_17_Solution.png

![PS_17_Solution.png](files/PS_17_Solution.png)

df = n - 1 = 6 - 1 = 5

alpha = 5% = 0.05

The critical value for χ2 at 0.05 is 11.07

As χ2 < χ2 (Critical), null hypothesis (H0) should not be rejected. We failed to reject H0.

## Problem Statement 18:

In a certain town, there are about one million eligible voters. A simple random sample of
10,000 eligible voters was chosen to study the relationship between gender and
participation in the last election. The results are summarized in the following 2X2 (read
two by two) contingency table:

#### files/PS_18.png

![PS_18.png](files/PS_18.png)

We would want to check whether being a man or a woman (columns) is independent of
having
voted in the last election (rows). In other words, is “gender and voting independent”?

### Solution:

H0: Sex is independent of voting.

H1: Sex and voting are dependent.

#### files/PS_18_Solution_1.png

![PS_18_Solution_1.png](files/PS_18_Solution_1.png)

##### files/PS_18_Solution_2.png

![PS_18_Solution_2.png](files/PS_18_Solution_2.png)

df = (rows - 1) * (cols - 1) = (2-1) * (2-1) = 1

The critical value of χ2 at 5% level of significance when df = 1 is 3.84.

χ2 > χ2 (critical) => 6.584 > 3.84

So we reject the null hypothesis (H0).

## Problem Statement 19:

A sample of 100 voters are asked which of four candidates they would vote for in an
election. The number supporting each candidate is given below:

Higgins Reardon White Charlton

41      19      24      16

Do the data suggest that all candidates are equally popular? [Chi-Square = 14.96, with
3 df, 𝑝̂ < 0.05].

### Solution:

H0: All candidates are equally popular.

H1: All candidates are not equally popular.

#### files/PS_19_Solution_1.png

![PS_19_Solution_1.png](files/PS_19_Solution_1.png)

df = n - 1 = 4 - 1 = 3

The critical value of Chi-Square for a 0.05 significance level and 3 d.f. is 7.82.

χ2 > χ2 (critical) => 14.96 > 7.82

So we reject null hypothesis (H0)

## Problem Statement 20:

Children of three ages are asked to indicate their preference for three photographs of
adults. Do the data suggest that there is a significant relationship between age and
photograph preference? What is wrong with this study? [Chi-Square = 29.6, with 4 df: 𝑝̂ <
0.05].

#### files/PS_20.png

![PS_20.png](files/PS_20.png)

### Solution:

H0: There is no significance relationship between age and photography.

H1: There is a significance relationship between age and photography.

#### files/PS_20_Solution.png

![PS_20_Solution.png](files/PS_20_Solution.png)

df = (rows - 1) * (cols - 1) = (3-1) * (3-1) = 2 * 2 = 4

The critical value of χ2 at 1% level of significance when df = 4 is 13.28.

χ2 > χ2 (critical) => 29.6032 > 13.28

So we reject the null hypothesis (H0).

## Problem Statement 21:

A study of conformity using the Asch paradigm involved two conditions: one where one
confederate supported the true judgement, and another where no confederate gave the
correct response.

#### files/PS_21.png

![PS_21.png](files/PS_21.png)

### Solution:

H0: There are no significant difference between the "support" and "no support" conditions

H0: There are significant difference between the "support" and "no support" conditions

#### files\PS_21_Solution.png

![PS_21_Solution.png](files\PS_21_Solution.png)

df = (rows - 1) * (cols - 1) = (2-1) * (2-1) = 1 * 1 = 1

The critical value of χ2 at 1% level of significance when df = 1 is 6.63.

χ2 > χ2 (critical) => 18.10345 > 6.63

So we reject the null hypothesis (H0).

## Problem Statement 22:

#### files\PS_22.png

![PS_22.png](files\PS_22.png)

### Solution:

H0: There is no relationship between height and leadership qualities.

H1: There is a relationship between height and leadership qualities.

#### files\PS_22_Solution.png


![PS_22_Solution.png](files\PS_22_Solution.png)

df = (rows - 1) * (cols - 1) = (3-1) * (2-1) = 2 * 1 = 2

The critical value of χ2 at 1% level of significance when df = 2 is 9.21

χ2 > χ2 (critical) => 10.9881 > 9.21

So we reject the null hypothesis (H0).

## Problem Statement 23:

Each respondent in the Current Population Survey of March 1993 was classified as
employed, unemployed, or outside the labor force. The results for men in California age
35-44 can be cross-tabulated by marital status, as follows:

#### files\PS_23.png

![PS_23.png](files\PS_23.png)

Men of different marital status seem to have different distributions of labor force status.
Or is this just chance variation? (you may assume the table results from a simple random
sample.)

### Solution:

H0: There is no relationship between marital status and labour status.

H1: There is a relationship between marital status and labour status.

#### files\PS_23_Solution.png

![PS_23_Solution.png](files\PS_23_Solution.png)

df = (rows - 1) * (cols - 1) = (3-1) * (3-1) = 2 * 2 = 4

The critical value of χ2 at 1% level of significance when df = 4 is 13.28

χ2 > χ2 (critical) => 30.9587 > 13.28

So we reject the null hypothesis (H0).